apps/meeting/src/containers/MeetingProviderWrapper/index.tsx

```diff
@@ -26,24 +27,26 @@ const MeetingProviderWrapper: React.FC = () => {
 
   const getMeetingProviderWrapper = () => {
     return (
-      <MeetingProvider {...meetingConfigValue}>
```

Comment:
We do not want to change this right, but, add another method like `getMeetingProviderWrapperWithVF` say `getMeetingProviderWrapperWithBgBlur`. We want to test the demo such that we should be able to test demo with only bg blur, only VF, with VF and bgblur and without anything with simple controls without these features.